src/plone/testing/_z2_testbrowser.py

```diff
@@ -129,8 +27,8 @@ def saveState(func):
     """
     from AccessControl.SecurityManagement import getSecurityManager
     from AccessControl.SecurityManagement import setSecurityManager
-    from zope.component.hooks import getSite
-    from zope.component.hooks import setSite
+    from zope.site.hooks import getSite
+    from zope.site.hooks import setSite
```

Comment:
the imports from zope.site are deprecated: https://github.com/zopefoundation/zope.site/blob/master/src/zope/site/hooks.py